sockeye/coverage.py

```diff
@@ -19,23 +19,27 @@
 
 import mxnet as mx
 
+from sockeye.layers import LayerNormalization
+
 logger = logging.getLogger(__name__)
 
 
 def get_coverage(coverage_type: str,
-                 coverage_num_hidden: int) -> 'Coverage':
+                 coverage_num_hidden: int,
+                 layer_normalization: bool) -> 'Coverage':
     """
     Returns a Coverage instance.
 
     :param coverage_type: Name of coverage type.
     :param coverage_num_hidden: Number of hidden units for coverage vectors.
+    :param layer_normalization: If true, applies layer normalization to ActivationCoverage.
     :return: Instance of Coverage.
     """
 
     if coverage_type == "gru":
         return GRUCoverage(coverage_num_hidden)
```

Comment:
Sure. The only problem is to do that in a backwards compatible manner in the future, because now if layer_normalization is turned on, the behavior is to not use it in GRUCoverage, so we shouldn't turn LN on for GRUCoverage in the future. I guess one way would be to add another coverage type like "gruln", but it might be confusing that layer_normalization in general only covers some aspects of the network and not others.